website/docs/hms.md

```diff
@@ -127,6 +127,16 @@ cd $HUDI_HOME/hudi-sync/hudi-hive-sync
 --table <tableName>
 ```
 
+```shell md title="example for s3 with HMS"
+./run_sync_tool.sh  \
+--metastore-uris 'thrift://hive-metastore:9083' \
+--partitioned-by city \
```

Comment:
`<partition_field>` ?

Reply:
Partition Field isn't an option in https://hudi.apache.org/docs/syncing_metastore/

Reply:
I think it's partitioned-by per https://hudi.apache.org/docs/syncing_metastore/